### PR TITLE
First steps for #1838-Photos in the favourites collection are not dis…

### PR DESCRIPTION
**Pull request by sauravvishal8797**
…played with star mark.

First step for issues #1838 

Changes: Photos in the favourites collection are not displayed with star mark. The photos which have been added to the favourites collection are displayed with star mark only in their parent album in Local/Hidden folders.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/adapters/MediaAdapter.java [hunk 1]
```diff
@@ -35,6 +36,7 @@
 public class MediaAdapter extends RecyclerView.Adapter<MediaAdapter.ViewHolder> {
 
     private ArrayList<Media> medias;
+    private boolean fav=false;
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/adapters/MediaAdapter.java]: Pls add a space between = and lhs and rhs

**Comment by sauravvishal8797** [comment]
@abishekvashok Did the changes. Please review.

**Comment by sauravvishal8797** [comment]
@abishekvashok @isuruAb @anantprsd5 Please review.

**Comment by sauravvishal8797** [comment]
@abishekvashok Please review this one.

**Pull request merged**